lib/pages/projects/project.dart

```diff
@@ -8,8 +8,8 @@ class Project {
   static final dbColorCode = "colorCode";
   static final dbColorName = "colorName";
 
-  int id, colorValue;
-  String name, colorName;
+  int? id, colorValue;
+  String? name, colorName;
```

Comment:
`name`, `colorValue` and `colorName` are non-nullable. We can avoid `String?` here. This is will also help us to  remove `!` force null conversion.